spec.bs

```diff
@@ -42,6 +42,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/C
     text: create an agent; url: create-an-agent
     text: immediately; url: immediately
     text: valid floating-point number; url: valid-floating-point-number
+    text: structured clone algorithm; url: safe-passing-of-structured-data

```

Comment:
This is a weird section to reference. Can we just reference https://html.spec.whatwg.org/multipage/structured-data.html#dom-structuredclone, which is the actual algorithm directly?

Reply:
Done